cinema_service/urls.py

```diff
@@ -1,11 +1,35 @@
-from django.conf import settings
-from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView
+)
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
```

Comment:
Ensure that the `cinema/urls.py` file defines an `app_name` variable as 'cinema' to support the use of the `namespace` argument here.